### ISSUE TITLE
Table Visualization distortion of big numbers on UInt64 type

### ISSUE THREAD

**Issue by Nopius** [getredash/redash#6159]
### Issue Summary

Bug appers with clickhouse data source and big numbers with UInt64 data type. Problem is proved to be on frontend/Visualizer side, because returned XHR is in correct format and with correct meta.

### Steps to Reproduce

1. Create clickhouse data source.
2. Create query in clickhouse data source 'select 8722821678000304785'
3. Execute query with network debug turned on
4. Check network resonse:
```
{
    "query_result": {
        "id": 7932109,
        "query_hash": "38c14fd87388f3b88a159d215bfac6c8",
        "query": "select 8722821678000304785 LIMIT 1000",
        "data": {
            "columns": [
                {
                    "name": "8722821678000304785",
                    "friendly_name": "8722821678000304785",
                    "type": "integer"
                }
            ],
            "rows": [
                {
                    "8722821678000304785": 8722821678000304785
                }
            ]
        },
        "data_source_id": 67,
        "runtime": 0.4866828918457031,
        "retrieved_at": "2023-07-10T01:20:52.933Z"
    }
}
```
5. Check visualizated value (wrong): 8,722,821,678,000,305,152
![image](https://github.com/getredash/redash/assets/5144107/01978145-69d9-4504-89e6-5826c5ef23d7)

When saving in CSV/TSV the values are correct, the problem appears only in a browser.

### Technical details:

* Redash 10/Linux
* Chrome/Windows:
* How did you install Redash: docker image, runs in k8s

**Comment by justinclift** [comment]
I remember running into something very like this a while back, and I *think* from rough memory it's this:

https://discuss.redash.io/t/a-numeric-column-in-sql-shows-different-value-than-it-actually-contains-in-redash-display/9587/14

**Comment by justinclift** [comment]
There is a suggested way forward for fixing it too, we'd just need someone to do the implementation.  Might not even be too hard either. :smile:

**Comment by justinclift** [comment]
Yeah.  We'd need to figure out a "better" approach to solve this.  It *might* be a substantial piece of work, or could turn out to not be.  No idea personally. :smile:

**Comment by justinclift** [comment]
@eradman Would this be of use for solving the problem?

https://github.com/MikeMcl/bignumber.js

**Comment by eradman** [comment]
The effort to solving this would be huge and there is very little benefit. People cannot read a large numbers, hence a number such as `87228216780304785` will nearly _always_ formatted in a human readable form, such as `77.47 PiB`.

This is surprising at first, but since this is how JavaScript numbers work, it is not unique to Redash. Also CSV/TSV export is generated in Python without a loss of precision.

**Comment by mirkan1** [comment]
There must be a better way to force server to send numbers as strings, I already patched numeral.js in order to work on big numbers

**Comment by justinclift** [comment]
> People cannot read such large numbers ...

Ahhh.  We have a customer affected by the incorrect display of large values.

They're working around it currently by casting it to a string for the text representation, but they'd like it fixed at some point.

When I have spare time to properly get up to speed with this part of things I should have a go at fixing this. :smile:
